ipaserver/install/upgradeinstance.py

```diff
@@ -161,6 +163,29 @@ def __save_config(self):
             else:
                 self.backup_state('nsslapd-global-backend-lock', global_lock)
 
+        with open(self.filename, "r") as in_file:
+            parser = GetEntryFromLDIF(in_file, entries_dn=[COMPAT_DN])
+            parser.parse()
+
+            try:
+                compat_entry = parser.get_results()[COMPAT_DN]
+            except KeyError:
+                pass
+            else:
+                try:
+                    schema_compat_enabled = compat_entry[
+                        'nsslapd-pluginEnabled'][0].decode('utf-8')
```

Comment:
Is the returned entry just a normal dictionary, does python-ldap not set case for all the keys, or doesn't python-ldap have their own caseless dictionary for these cases?